source/lib-intro.tex

```diff
@@ -1614,15 +1614,15 @@
 \begin{codeblock}
 #include <utility>
 
-// Requires: \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
+// Mandates: \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
```

Comment:
Shouldn't we then also get rid of "shall", since the "mandates" style implies that?

Reply:
Yes, a Mandates condition is just stated. "is swappable with".